packages/aws-cdk/lib/cli.ts

```diff
@@ -249,6 +250,20 @@ async function initCommandLine() {
   });
   await configuration.load();
 
+  if (argv.failOnNotices) {
+    const noticeProps = {
+      outdir: configuration.settings.get(['output']) ?? 'cdk.out',
+      acknowledgedIssueNumbers: configuration.context.get('acknowledged-issue-numbers') ?? [],
+    };
+
+    const notices = await getApplicableNotices(noticeProps);
```

Comment:
How does this interact with prefetching, or fetch errors? 

Feels like if you explicitly request this, we should fail hard if we can't retrieve the notices, right?

Reply:
Hmm, I'm not sure. I can see arguments for both sides:

* For failing hard: if you selected this option, it means you don't want to take any chances. So if there is the _possibility_ of a notice applying to you (but you can't know because the fetch failed), we should kill the execution right away.
* For skipping: in general, this feature operates on a sort of best-effort delivery. So if you're running the CLI interactively and it fails to fetch the notices, you won't know about it and therefore you won't be able to respond. This is acceptable in interactive mode, so it should be acceptable in automatic mode as well.

Reply:
I don't understand what you mean with "for skipping". I'm *willing* to go along with silently failing if we can't fetch notices... but it's making me uneasy.